renovate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [ "github>redhat-developer/app-services-renovate-config" ],
```

Comment:
Note: That has no effect as there are no python specific rules. If you intend adding some rules please remember to add them to config.

Reply:
great catch. I will put it on my todo 👍  🙇‍♂️